enacts/enactstozarr.py

```diff
@@ -7,21 +7,25 @@
 from pathlib import Path
 import pingrid
 import pandas as pd
+from functools import partial
 
 
 CONFIG = pingrid.load_config(os.environ["CONFIG"])
 
-ZARR_RESOLUTION = CONFIG["zarr_resolution"]
-input_path, output_path, var_name = CONFIG['vars'][variable]
-
-def set_up_dims(xda):
+def set_up_dims(xda, time_res="daily"):
     
     datestr = Path(xda.encoding["source"]).name.split("_")[2]
-    xda = xda.expand_dims(T = [dt.datetime(
-      int(datestr[0:4]),
-      int(datestr[4:6]),
-      int(datestr[6:8])
-    )])
+    year = int(datestr[0:4])
+    month = int(datestr[4:6])
+    if time_res == "daily":
+        day = int(datestr[6:8])
+    elif time_res == "dekadal":
+        day = (int(datestr[6:7]) - 1) * 10 + 1
```

Comment:
I've only picked this up from what either Kevin or you (I think it's either Kevin or you) wrote to convert dekadal files. I haven't given it more thought for this PR.

Reply:
OK. And I guess it's consistent with how we're handling the daily resolution, since y, m, d implicitly means 0:00 (start of the say), not 12:00 (midpoint). Making an issue to think about this more later.